lib/modules/manager/pub/artifacts.spec.ts

```diff
@@ -19,7 +19,9 @@ process.env.CONTAINERBASE = 'true';
 const lockFile = 'pubspec.lock';
```

Comment:
The PR title has a style error: `add to extract` is not following the grammar rules:

> feat(manager/pub): add to extract flutter as a dependency 

Reading the PR body text, it looks like a correct PR title should be something like this?

> feat(manager/pub): extract Flutter SDK